services/email_verification.js

```diff
@@ -35,45 +41,57 @@ const mailer = require("nodemailer").createTransport({
 });
 
 const server = micro(async req => {
-    const data = await json(req);
+    const data = await json(req.body);
     // generate a token, add it to redis
     let id = idGen.nextId();
-    client.hset(data.user.name, `email_verification`, JSON.stringify({
-        deleteThis: true,
-        id
-    }), async err => {
+    let token = bcrypt.hashSync(`${data.user.email}:${id}`)
```

Comment:
Probably change `id` and `token` to consts, since they don't change, and the above `data` is a const.